data/seedDataGenerator.js

```diff
@@ -0,0 +1,128 @@
+const modelautomotive = ['parts', 'interior accessories', 'tools & equipment', 'car care', 'wheels'];
+const modelgrocery = ['Wine - Toasted Head', 'Bacardi Mojito', 'Crab - Claws', 'Danishes - Mini Cheese', 'Breakfast Quesadillas', 'Icecream', 'Shrimp - 21/25, Peel And Deviened', 'Beef - Eye Of Round', 'Figs', 'Bread - White, Unsliced', 'Cheese - Parmesan Cubes', 'Potatoes - Mini White 3 Oz', 'Flower - Potmums', 'Chips Potato All Dressed - 43g', 'Versatainer Nc - 888', 'Bacon Strip Precooked', 'Crackers - Water', 'Veal - Tenderloin, Untrimmed', 'Salt - Seasoned', 'Beef - Outside, Round', 'Cassis', 'Caviar - Salmon', 'Cheese - Feta', 'Barley - Pearl', 'Longos - Chicken Curried', 'Muffins - Assorted'];
+const modelbeauty = ['Makeup', 'Skin Care', 'Hair Care', 'Fragrance', 'Foot, Hand & Nail Care', 'Tools & Accessories', 'Shave & Hair Removal', 'Personal Care', 'Oral Care'];
+const modelbook = ['Eoin Dempsey', 'Caroline Mitchell', 'Christopher Rice', 'Rhys Bowen', 'Lisa Wingate', 'Matthew Wade', 'Hugh Acheson', 'Roger Priddy', 'Margaret Wise Brown, Clement Hurd', 'Eric Carle', 'Giles Andreae, Guy Parker-Rees'];
+
+// shipping property
+const shipping = ['Free Shipping', 'Regular Shipping', 'Same-Day Delivery', 'One Day Shipping'];
+
+// inStock property
+const stock = [true, false];
+
+// description property
+const description = ['Lorem ipsum dolor sit amet, consectetur adipiscing elit.', 'Sed tempus odio nec neque porta, nec sagittis lorem ullamcorper.', 'Sed nec odio lacinia, rhoncus sapien letius, bibendum sapien.',
+  'Cras ut sem vel lectus rhoncus imperdiet a at leo.', 'Maecenas congue leo at tempor lacinia.', 'Morbi nec felis sagittis, pharetra lorem quis, pretium nulla.', 'Quisque efficitur leo sed lacus egestas sagittis.', 'Donec eu lacus sed lacus elementum bibendum non sed magna.', 'Etiam a risus luctus, lacinia dui in, varius lorem.', 'Etiam ut augue sed justo eleifend ullamcorper.', 'Sed ut dolor in nibh laoreet posuere non quis nunc.', 'Vestibulum id odio vel turpis dictum commodo.', 'Nullam id metus imperdiet, bibendum urna condimentum, aliquet elit.', 'Suspendisse sit amet erat vitae quam vestibulum finibus.'];
+
+// random index generator
+const randomElement = (array) => {
+  const randomIndex = Math.floor(Math.random() * array.length);
+  return array[randomIndex];
+};
+
+// random integer between two values
+const getRandomInt = (min, max) => {
+  const minInt = Math.ceil(min);
+  const maxInt = Math.floor(max);
+  return Math.floor(Math.random() * (maxInt - minInt)) + minInt;
+};
+
+// random integer
+const getOneRandomInt = max => Math.ceil(Math.random() * Math.ceil(max));
+
+// return an array of random description
+const randomDescription = () => {
+  const result = [];
+  while (result.length < getOneRandomInt(20)) {
+    result.push(randomElement(description));
+  }
+  return result;
+};
+
+// data generator
+const mapping = arr => arr.map((item) => {
```

Comment:
The code makes sense. The only issue is that it seems like you are repeating yourself in certain ways, however, I can see that as a result of making different brands and models. If you had time, and didn't care about how the data looked, you could just use lorem ipsum to make a small general list and use it across the board. It will definitely shorten your code, and make it easier to follow.